### ISSUE TITLE
[BUG] Token Action Hud does not display spells or features

### ISSUE THREAD

**Issue by gioppoluca** [PwQt/magic-items-2#176]
**Module Version:** v4.2.1

**Describe the bug**
In versions previous V12 and dnd5e 3.3.1 the spells/features from magic items were listed in Token Action Hud, now do not shows up

**Expected behavior**
Spells/features from magic items get listed in Token Action Hud
**Browser:**
 - Foundry app

**Foundry Version:** v12.330

**Game System:** dnd5e 3.3.1

**Additional context**
midi, dfred, J&B, AA, Gambit, Chris, dice so nice, ...

**Comment by PwQt** [comment]
https://github.com/Larkinabout/fvtt-token-action-hud-dnd5e/issues/85#issuecomment-2039632207 have you done this?

**Comment by gioppoluca** [comment]
Yes ... after your comment ... but it seems that the latest release do not show the menu even if added manually.
